src/styles/sidebar.styl

```diff
@@ -24,7 +24,7 @@
 
     // need this due to no css-modules usage in this app
     aside
-        width   100%!important
+        width   100%!important // @stylint ignore
```

Comment:
@gregorylegarec I knew about this one, i gave my green light on it as long as there's no CSS-modules. But if it stays after introducing CSS-modules, i'm gonna bite and i'm gonna bite very hard. 😆